### PR TITLE
Move margin to the right place.

### PR DESCRIPTION
**Pull request by teaP**
Ew bad:
image

Whew all better now:
image

### CODE REVIEW

File: dev/CommonStyles/CommandBar_themeresources.xaml [hunk 1]
```diff
@@ -830,6 +829,7 @@
                                 <!-- Use a ContentControl rather than a ContentPresenter so that IsEnabled can be set to false
                                      in the Minimal/HiddenClosed states to remove it from being a tab-stop candidate. -->
                                 <ContentControl x:Name="ContentControl"
+                                      Margin="{TemplateBinding Padding}"
```
Comment (chingucoding) [inline dev/CommonStyles/CommandBar_themeresources.xaml:832]: Should we backport those changes to the V1 style?

Also, great PR description, definitely made me chuckle 😄

**Review by StephenLPeters — APPROVED** [review]

File: dev/CommonStyles/CommandBar_themeresources.xaml (hunk 1 above)
Comment (StephenLPeters) [reply dev/CommonStyles/CommandBar_themeresources.xaml:832]: good question

Comment (beervoley) [reply dev/CommonStyles/CommandBar_themeresources.xaml:832]: @teaP I don't understand how did TemplateBinding to 4,0,0,0 margin make such a change in the visuals?

Comment (mdtauk) [reply dev/CommonStyles/CommandBar_themeresources.xaml:832]: I suspect that is not ensuring the same settings for the before and after screenshots.  The label position is a setting you can change.

**Pull request merged**